omegaconf/omegaconf.py

```diff
@@ -398,41 +404,114 @@ def unsafe_merge(
 
         return target
 
-    @staticmethod
-    def _tokenize_args(string: Optional[str]) -> List[str]:
-        if string is None or string == "":
-            return []
-
-        def _unescape_word_boundary(match: Match[str]) -> str:
-            if match.start() == 0 or match.end() == len(match.string):
-                return ""
-            return match.group(0)
-
-        escaped = re.split(r"(?<!\\),", string)
-        escaped = [re.sub(r"(?<!\\) ", _unescape_word_boundary, x) for x in escaped]
-        return [re.sub(r"(\\([ ,]))", lambda x: x.group(2), x) for x in escaped]
-
     @staticmethod
     def register_resolver(name: str, resolver: Resolver) -> None:
+        # TODO re-enable warning message before 2.1 release (see also test_resolver_deprecated_behavior)
+        # warnings.warn(
+        #     dedent(
+        #         """\
+        #     register_resolver() is deprecated.
+        #     See https://github.com/omry/omegaconf/issues/426 for migration instructions.
+        #     """
+        #     ),
+        #     stacklevel=2,
+        # )
+        return OmegaConf.legacy_register_resolver(name, resolver)
+
+    # This function will eventually be deprecated and removed.
+    @staticmethod
+    def legacy_register_resolver(name: str, resolver: Resolver) -> None:
         assert callable(resolver), "resolver must be callable"
         # noinspection PyProtectedMember
         assert (
             name not in BaseContainer._resolvers
-        ), "resolved {} is already registered".format(name)
+        ), "resolver {} is already registered".format(name)
```

Comment:
If you are already changing it, can you fix to use an fstring?

Reply:
Trivial fix in 5873bc0 => resolving